git.go

```diff
@@ -38,6 +38,11 @@ func MustGitCommit(format string, a ...interface{}) {
 	fmt.Printf("\033[0m")
 }
 
+// MustGitRemove runs a "git rm" on the provided file path, relative to GIT_REPO.
+func MustGitRemove(relativePath string) {
+	MustRunGitCmd("rm", relativePath)
```

Comment:
not sure adding a function wrapper adds anything, probably best to call `MustRunGitCmd("rm", relativePath)` on its own